lib/Mojolicious/Command/get.pm

```diff
@@ -16,7 +16,8 @@ sub run {
   my ($self, @args) = @_;
 
   # Data from STDIN
-  vec(my $r, fileno(STDIN), 1) = 1;
+  my $r = '';
+  vec($r, fileno(STDIN), 1) = 1;
```

Comment:
Why not `vec(my $r = '', fileno(STDIN), 1) = 1;`?

Reply:
That would work also

Reply:
done